ITISFoundation/osparc-simcore
# is743/service selection

## What do these changes do?

Delivers a new version of the service selection modal view.
- [x] Each service card shows: title, description and author/contact
- [x] Auto focus in the Filter textfield when opening the view
- [x] Enter key stroke automatically adds
- [x] Tab key moves the selected service
- [x] Sorting of the entries (alphabetically)

## Related issue number
#743 

## Screenshot
![image](https://user-images.githubusercontent.com/4764217/57761226-608a7180-76fd-11e9-8c9b-2135f462d308.png)


## Test it 
* Use the window controls
* Create new services (double click or add button)
* Check versions are working
* Use filters
* Check design (the current one is just a proposal)

## Checklist
